manage/checklist.rst

```diff
@@ -0,0 +1,51 @@
+Checklist
+=========
+
+General
+-------
+
+The checklist feature allows agents to keep track of the tasks to be completed.
+It is activated by default and shows up in another tab in the right sidebar
+in the ticket view (see :user-docs:`here </extras/checklist.html>` what it
+looks like from agent's perspective).
+
+.. figure:: /images/manage/checklist/checklist-settings.png
+   :alt: Screenshot showing checklist feature in admin area.
+
+Usage
+-----
+
+To activate/deactivate the feature, simply switch the toggle labeled with
+"Enable Checklists" at the top of the page.
+
+You can optionally create one or more templates with predefined checklist
+items. For this, just click on the "New Checklist Template" button and add
+the checklist items. After submitting the form, the template can be used in
+tickets.
+
+You can clone or delete the template by clicking on the ︙ button and select
+the corresponding action.
+
+To edit the template, simply click on the title of the template. This opens
+a dialog where you can edit the checklist and the items (add, rename,
+reorder, delete them) and set the entire template to active/inactive.
+
+.. figure:: /images/manage/checklist/checklist-template-edit.png
+   :alt: Screenshot showing the checklist template edit dialog.
+   :align: center
+   :scale: 70%
+
+Remarks
+-------
+
+- It is only possible to add one checklist per ticket.
+- There is no specific permission for the checklist feature. You can turn it off
+  or on and provide optionally usable checklist templates.
```

Comment:
Perhaps add something along the lines:

`If the agent has access to the ticket, they will also have access to its checklist.`